src/misc/unicode.cpp

```diff
@@ -629,12 +875,13 @@ static void dos_to_wide(const std::string& str_in,
 		const auto byte = static_cast<uint8_t>(character);
 		if (GCC_UNLIKELY(byte >= decode_threshold_non_ascii)) {
 			// Character above 0x07f - take from code page mapping
+			auto& mappings = per_code_page_mappings[code_page];
 
-			if (!mappings_reverse_by_codepage.count(code_page) ||
-			    !mappings_reverse_by_codepage[code_page].count(byte)) {
+			if (!per_code_page_mappings.count(code_page) ||
+			    !mappings.grapheme_to_dos.count(byte)) {
```

Comment:
Suggest quantity comparisons (count == 0)

Reply:
OK, changed - everywhere in this file.